tests/service/test_rest_api.py

```diff
@@ -110,9 +110,7 @@ def test_get_plan_with_device_reference(handler: Handler, client: TestClient) ->
                         "title": "Delay",
                     },
                     "detectors": {
-                        "items": {
-                            "_detectors": "<class " "'bluesky.protocols.Readable'>"
-                        },
+                        "items": {"type": "<class " "'bluesky.protocols.Readable'>"},
```

Comment:
I think long term we probably want `"type": "bluesky.protocols.Readable"`, such that we can `GET /devices?type=bluesky.protocols.Readable`